src/__tests__/patients/patient-slice.test.ts

```diff
@@ -130,6 +237,7 @@ describe('patients slice', () => {
     })
   })
 
+  // should check for the Toast
```

Comment:
yeah. I added a toast for updating a Patient and made it redirect history to `/patients/${newPatient.id}`, since saving on Edit Patient ought to redirect to that. as a side effect of that, adding a new Related Person now moves you off the Related Persons tab and back onto the General Information tab, but I'm not sure if should do anything else about that now.

Reply:
added this test